tests/test_utils/test_path_helpers.py

```diff
@@ -0,0 +1,196 @@
+"""Tests for path_helpers.py """
+import pytest
+
+from src.utils.path_helpers import (
+    get_paths,
+    create_staging_config,
+    create_ni_staging_config,
+    create_mapping_config,
+    create_module_config,
+    update_config_with_paths,
+)
+
+
+@pytest.fixture(scope="module")
+def config():
+    config = {
+        "global": {"network_or_hdfs": "network"},
+        "network_paths": {
+            "root": "R:/DAP_emulation/",
+            "snapshot_path": "snapshot_path/snap.csv",
+            "secondary_snapshot_path": "secondary_snapshot_path/snap2.csv",
+            "postcode_masterlist": "postcode_masterlist_path/postcode.csv",
+            "ni_full_responses_path": "03_northern_ireland/2021/TEST_ni.csv",
+            "manual_imp_trim_path": "06_imputation/man_trim/trim_qa.csv",
+            "manual_outliers_path": "07_outliers/man_out/man_out.csv",
+        },
+        "years": {"survey_year": 2022},
+        "staging_paths": {
+            "folder": "01_staging",
+            "feather_output": "feather",
+        },
+        "ni_paths": {
+            "folder": "03_northern_ireland",
+            "ni_staging_output_path": "ni_staging_qa",
+        },
+        "construction_paths": {
+            "folder": "02_construction",
+            "qa_path": "construction_qa",
+        },
+        "2022_mappers": {
+            "mappers_version": "v1",
+            "postcodes_mapper": "pcodes_2022.csv",
+            "itl_mapper_path": "itl_2022.csv",
+        },
+        "imputation_paths": {
+            "folder": "06_imputation",
+            "qa_path": "imputation_qa",
+            "manual_trimming_path": "manual_trimming",
+        },
+        "outliers_paths": {
+            "folder": "07_outliers",
+            "qa_path": "outliers_qa",
+            "auto_outliers_path": "auto_outliers",
+        },
+        "estimation_paths": {
+            "folder": "08_estimation",
+            "estimation_qa": "estimation_qa",
+        },
+        "apportionment_paths": {
+            "folder": "09_apportionment",
+            "qa_path": "apportionment_qa",
+        },
+        "outputs_paths": {
+            "folder": "10_outputs",
+            "qa_path": "outputs_qa",
+        },
+    }
+    return config
```

Comment:
this doesn't really seem appropriate. can we use a smaller test fixture?

Reply:
It's needed for the update_config_with_paths function at the end, but I've simplified this by not hard-coding the list of modules.